math/median.go

```diff
@@ -0,0 +1,37 @@
+// author(s) [jo3zeph](https://github.com/jo3zeph)
+// description: Find the median from a set of values
+// see median_test.go
+
+package math
+
+import (
+	"golang.org/x/exp/constraints"
+	"golang.org/x/exp/slices"
+)
+
+type Data interface {
+	constraints.Float | constraints.Integer
+}
```

Comment:
We already have the `constraints` [here](https://github.com/TheAlgorithms/Go/blob/master/constraints/contraints.go) and we also have all the sorting algorithms, you can find the usage [here](https://github.com/TheAlgorithms/Go/blob/master/sort/sorts_test.go)

Please refrain from using the `/x/` packages since they are subject to changes meaning they can break our repo builds even when we change the go version.

Reply:
I see, I have already amended the package imports

Reply:
I can still see those imports, you probably haven't committed your changes or pushed your changes.

Reply:
Also, we have the [`constraints.Number`](https://github.com/TheAlgorithms/Go/blob/16c92748a3db40d1c6f916e390342d02e21c22d0/constraints/contraints.go#L28-L30) present in our constraints package. You should use that instead of creating a new `Data` constraint, which is exactly the same.

Reply:
My bad, i have already removed it